### ISSUE TITLE
Arrows problem in 1.8

### ISSUE THREAD

**Issue by Playerliqht** [ProtocolSupport/ProtocolSupport#1289]
Server version : 1.16.3 (PAPER)
/ps info : 
![Screenshot_1](https://user-images.githubusercontent.com/41956661/95688615-c2f5e300-0c0b-11eb-95b4-80992fe4a827.png)
https://github.com/ProtocolSupport/ProtocolSupport/commit/90d18049e42cb266c0db13c6cf8d8441b9db57a7)

In **version 1.8** of minecraft:

- **The particles of the arrow shot from the bow**
- **Some arrows cannot be seen when shot by other players!**
- **Arrows entities are sometimes invisible to the client 1.8.**

![Screenshot_2](https://user-images.githubusercontent.com/41956661/95688619-c5583d00-0c0b-11eb-82b3-5536b9b3026a.png)

Thank you o/

**Comment by Playerliqht** [comment]
@Shevchik This is the same for **splash potions particles in 1.8** !

**Comment by ludgart** [comment]
Same problem with the version 1.12.2 on the server 1.16.5-paper. ProtocolSupport: 594

**Comment by Playerliqht** [comment]
> Same problem with the version 1.12.2 on the server 1.16.5-paper. ProtocolSupport: 594
> 

A small problem, but important for pvp players!

**Comment by Playerliqht** [comment]
Hi ! I put a little message to know if this problem can be updated quickly. It's really a small detail but most people who play find it so important, especially for pvp players. 
Yours, Shev, hope you have the time! thanks again

**Comment by Shevchik** [comment]
Making particles work is generally not easy, since some of them changed way too much.
Not sure about invisible arrows tho, i couldn't reproduce it myself, any specifics on how exactly does it happen?

**Comment by Playerliqht** [comment]
> Making particles work is generally not easy, since some of them changed way too much.
> Not sure about invisible arrows tho, i couldn't reproduce it myself, any specifics on how exactly does it happen?

In 1.8, when shooting an arrow with the bow fully loaded, we normally have an arrow particle effect (detail).

Also, in 1VS1 combat, the attacker who shoots an arrow will have his arrow go but not the player who is against him. (Sometimes)

There is also a potions problem which is very, very unpleasant: some potions do not splash and if some do, they are almost never the right "splash potion colors".

To sum up, most particle issues are in 1.8.X

I hope I could enlighten you a little more on the subject! Thank you

**Comment by Shevchik** [comment]
The latest commit https://github.com/ProtocolSupport/ProtocolSupport/commit/f8e98a1f39e2796d45b65f1d3ce82f7f8f7392e2 might have fixed arrow invisibility issue, can you check?
